src/main/java/com/mindee/CLI.java

```diff
@@ -37,19 +39,6 @@ public class CLI {
     description = "Flag to set all words")
   boolean words;
 
-  @Option(names = {"-C",
```

Comment:
don't remove, but inverse the option, should be `"-c, --cut-pages"`, as by default we don't cut. When the option is activated, cut all pages after the 5th one.